company-profiles/joor.md

```diff
@@ -0,0 +1,43 @@
+# JOOR
+
+## Company blurb
+
+Redefining the wholesale landscape, JOOR is a global platform connecting brands and retailers with seamless, online transactions. With innovative solutions and a strong emphasis on user experience, we empower businesses to collaborate and trade effortlessly. At JOOR, it's not just about digitizing commerce – we’re reimagining it. 
+
+From iconic brands to up-and-coming retailers, we've been at the forefront, offering solutions that bridge the physical and digital, crafting the future of wholesale commerce.
+
+## Company size
+
+200ish dedicated professionals specializing in tech development, sales, customer support, marketing, and operations. The tech/engineering department is around 60 people.
+
+## Remote status
+
+The tech team is entirely remote. Most people based in Madrid, but there are employees in the US, Uruguai, Italy, Serbia, and other places. Other departments of the company can be on a hybrid model. 
+
+## Region
```

Comment:
The region in the readme as Europe but here it reads like is global? Can you confirm and match, please?

Reply:
Tbh, I don't know what I should put there. The team is global, but hiring right now is restricted to Europe. I put Europe in the readme because currently, they are not considering applications from outside Europe. Should I put global on readme?

Reply:
Valid point @gutogalego. I think I need to be more specific about this. I've always gone with "where the company hires" which is global for this entry but **right now** it's only Europe. let's keep this as Europe for now.